### DISCUSSION TITLE
Prompt browser is not safe, how to solve this problem?

### DISCUSSION THREAD

**Discussion by DarkFunct** [apify/fingerprint-suite#252] [Q&A]
I've tried login with Gmail or any Google services but it shows the following "This browser or app may not be secure" message:

![image](https://github.com/apify/fingerprint-suite/assets/38775458/4aad0dd9-710e-4f10-ad7a-93183fdf2be2)

**Comment by barjin** [comment]
Hello @DarkFunct and thank you for submitting this issue! (and sorry for getting to it this late)

I only had partial success with replicating this issue - sometimes, Google let me sign up just fine, and sometimes, I got the same error message as you did.

Most of the antibot scripts are based on some stochastic methods - calculating the conditional probability that the user is a bot from the browser / IP geolocation / OS / language / screen size etc. Because of this, you'll be getting better results if you're not trying to hide too many things at once. 

Regarding your (now edited out) example script, I would try to start with no evasions at all - set the fingerprint settings to whatever matches your actual setup as close as possible. Also, try disabling the proxy and use as few browser arguments as possible. This should give you a working base script, even though it might not work all the time either.

From here, try adding your evasions one by one - eg. first, try changing some of the `fingerprint-injector` options, then use proxy and work your way towards a solution that both works and is sufficiently "anonymous".

Let me know how did that help and / or feel free to ask any additional questions. 
Thanks!
